mongo/mongo.go

```diff
@@ -256,10 +263,21 @@ func (m *Mongo) checkoutConnection(server driver.Server) (conn driver.Connection
 	return conn, nil
 }
 
+func (m *Mongo) unacknowledgedRoundTrip(conn driver.Connection, req []byte) (err error) {
```

Comment:
I wonder if it would be cleaner to pass an `unacknowledged bool` argument to `roundTrip` rather than having a separate method, and passing the `unacknowledged` as a tag to any statsd calls (probably skipping the `response_size` distribution if `true`)?

Reply:
I avoided adding a boolean flag to `roundTrip` because the unacknowledged version doesn't return a response, but the stats stuff complicates that a bit, so I changed it to follow your suggestion in my latest commit.